marklogic-data-hub/src/main/java/com/marklogic/hub/DefaultHubConfigFactory.java

```diff
@@ -34,6 +34,9 @@ public HubConfig newHubConfig() {
         String prop = null;
         String mlUsername = getProperty("mlUsername");
         String mlPassword = getProperty("mlPassword");
+        
```

Comment:
All of the other properties do something like:

```java
prop = getProperty("...");
if (prop != null) {
  c.whatever = prop;
}
```

I would rather this be consistent with the other code.